addons/default/visiosoft/profile-module/resources/assets/js/education.js

```diff
@@ -0,0 +1,45 @@
+$(() => {
+    $.ajax({
+        url: '/ajax/getEducation',
+        success: ((res)=>{
+            $.each(res['education-part'], function (key, value) {
+                var selected = ""
+                if (res.user.education_part == value.id){ selected = 'selected'; }
+                $('#education_part').append('<option '+ selected +' value="'+ value.id +'">' + value.name + '</option>')
+            })
+        })
+    })
+
+    $('#education').on('change', () => {
+        $.ajax({
+            url: '/ajax/setEducation',
+            data: {
+                info: 'education',
+                education: $('#education').val()
+            },beforeSend: function (){
+                $('#education_part').html('');
+            },success: function (response) {
+                $('#education_part').html('<option>'+ choose_an_option +'</option>')
```

Comment:
'<option value="">'+ choose_an_option +'</option>' or else the value will be "choose an option"

Reply:
![resim](https://user-images.githubusercontent.com/22885454/102233503-6495fb00-3f01-11eb-8560-02e42b111ad1.png)

Reply:
`'<option value="">'+ choose_an_option +'</option>' or else the value will be "choose an option"`